codersdk/provisionerdaemons.go

```diff
@@ -287,11 +287,14 @@ func (c *wsNetConn) Close() error {
 	return c.Conn.Close()
 }
 
-// websocketNetConn wraps websocket.NetConn and returns a context that
+// WebsocketNetConn wraps websocket.NetConn and returns a context that
```

Comment:
Let's move this implementation to its own file, perhaps, doesn't make sense in the context of provisioner daemons only. Perhaps as a follow-up PR.